### ISSUE TITLE
How to generate train_annotations dataset for a custom image/class dataset?

### ISSUE THREAD

**Issue by kejitan** [divamgupta/image-segmentation-keras#184]
model.train(
    train_images =  "dataset1/images_prepped_train/",
    train_annotations = "dataset1/annotations_prepped_train/",
    checkpoints_path = "/tmp/vgg_unet_1" , epochs=5
)
In the above step, how do we generate train_annotations dataset for a custom images/class dataset? Do we have to label the classes by hand? 
Thank you?

**Comment by kejitan** [comment]
Sorry, I notice this has been addressed before. I close it
